src/wallet.cpp

```diff
@@ -118,8 +118,6 @@ bool CWallet::Unlock(const SecureString& strWalletPassphrase)
 
 bool CWallet::ChangeWalletPassphrase(const SecureString& strOldWalletPassphrase, const SecureString& strNewWalletPassphrase)
 {
-    bool fWasLocked = IsLocked();
```

Comment:
intentional ? windows build went in with this change here..think this was reverted by @justinvforvendetta

Reply:
I had removed that because it is unused.